source/Calamari/Kubernetes/Commands/KubernetesDeploymentCommandBase.cs

```diff
@@ -124,7 +124,7 @@ public override int Execute(string[] commandLineArguments)
 
             conventions.AddRange(CommandSpecificInstallConventions());
 
-            var runningDeployment = new RunningDeployment(pathToPackage, variables);
+            var runningDeployment = new RunningDeployment(pathToPackage, variables, GetEnvironmentVariables());
```

Comment:
I tried setting external EnvironmentVariables into the environment variables used by Kubectl but it didn't fix the `kubectl get/create namespace ...` commands.

Reply:
To be honest, I have no idea how `kubectl apply ...` does work if it has absolutely no environment variables set. Maybe it looks in a specific folder for config?

Reply:
We should probably look at setting ambient environment variables like this... maybe 🤔